leader-election-impl/src/main/java/com/palantir/paxos/PaxosAcceptorImpl.java

```diff
@@ -89,10 +113,14 @@ public PaxosPromise prepare(long seq, PaxosProposalId pid) {
 
     @Override
     public BooleanPaxosResponse accept(long seq, PaxosProposal proposal) {
+        leaderLog.debug("Asked to accept proposal for seq #{} with ID {} from node {}",
+                seq, proposal.getId().getNumber(), proposal.getId().getProposerUUID());
+
         try {
             checkLogIfNeeded(seq);
         } catch (Exception e) {
-            logger.error("log read failed for request: {}", seq, e);
+            leaderLog.error("log read failed for request: {} during accept phase; rejecting proposal", seq, e);
+            logger.error("log read failed for request: " + seq, e);
```

Comment:
I'm not sure I understand why we essentially log same thing twice
log format placeholders